src/app/errorReports/thanks/Thanks.jsx

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from 'react';
+import Button from '../../../components/button';
+import Heart from '../../../components/heart';
+import withErrorReportsForm from '../withErrorReportsForm';
+
+class Thanks extends Component {
+  constructor(props) {
+    super(props);
+    this.goToViewLocation = this.goToViewLocation.bind(this);
+  }
+
+  goToViewLocation() {
+    this.props.history.push(`/find/location/${this.props.match.params.locationId}/view`);
```

Comment:
The route is actually just `/find/location/${this.props.match.params.locationId}`, no "/view".
I don't think it actually works with "/view" at the end. If anything, it likely takes the user back to the map, which is what happens in general with URLs that end in unexpected patterns.

Reply:
Oh, good catch! The `/view` was taking me back to the location information page, since the Route was matching the start of the url (and `exact` wasn't being used), but it wasn't necessary.